build/webpack.config.js

```diff
@@ -123,9 +123,6 @@ const mainJsChunkConfig = buildMode => {
             options: babelConf(buildMode),
           },
         ],
-        exclude: [
```

Comment:
Shouldn't we let this one as this ?

Reply:
No, actually it's duplicated with the `ignore` rule in `babel.config.js`. If we let it, we have to pass the exception here also.